build.gradle

```diff
@@ -18,7 +18,9 @@ ext {
       kotlinStdLib: "org.jetbrains.kotlin:kotlin-stdlib:$kotlinVersion",
       kotlinReflect: "org.jetbrains.kotlin:kotlin-reflect:$kotlinVersion",
       kotlinGradlePlugin: "org.jetbrains.kotlin:kotlin-gradle-plugin:$kotlinVersion",
+      gradleNodePlugin: "com.moowork.gradle:gradle-node-plugin:1.0.0",
       junit: 'junit:junit:4.12',
+      spock: 'org.spockframework:spock-core:0.7-groovy-2.0',
```

Comment:
Nice, we can try to use the similiar `Speck` test framework for our unit tests in the compiler module